tests/integration/sharding_tests/application/charmcraft.yaml

```diff
@@ -0,0 +1,11 @@
+# Copyright 2023 Canonical Ltd.
```

Comment:
Feel free to ignore the test charm we copied over to `tests/integration/sharding_tests/application/`, this should be removed in the future